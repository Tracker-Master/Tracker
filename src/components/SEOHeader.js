import React from 'react';
import { Helmet } from 'react-helmet';

export const SEOHeader = ({ title }) => {
  return (
    <Helmet>
      <title>{title}</title>
      <meta property="og:title" content="Proyect Tracker" />
      <meta property="og:description" content="Proyect Tracker" />
      <meta property="og:image" content="" />
      <meta property="og:url" content=".github" />
      <meta property="og:site_name" content="Proyect Tracker" />
      <meta property="og:locale" content="es_EN" />
      <meta property="og:type" content="article" />
    </Helmet>
  );
};
