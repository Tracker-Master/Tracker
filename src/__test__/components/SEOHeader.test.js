import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import SEOHeader from '../../components/SEOHeader';

describe('<SEOHeader />', () => {
  
  const title = 'Titulo de la aplicaci√≥n';
  const component = <SEOHeader title={title} />;

  test('SEOHeader rendering ', () => {
    const seoHeader = shallow(component);
    expect(seoHeader.length).toBe(1);
  });

  test('SEOHeader snapshot', () => {
    const wrapper = create(component);
    expect(wrapper).toMatchSnapshot();
  });


});