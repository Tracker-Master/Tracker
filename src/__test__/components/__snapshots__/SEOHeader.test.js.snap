// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SEOHeader /> SEOHeader snapshot 1`] = `null`;
